data/core.yaml

```diff
@@ -1860,6 +1860,7 @@ en:
         message: "{feature} looks like a common feature with nonstandard tags"
         message_incomplete: "{feature} looks like a common feature with incomplete tags"
         reference: "Some features, for example retail chains or post offices, are expected to have certain tags in common."
+        nsi_reference: "The suggestions above are powered by the Name Suggestion Index."
```

Comment:
As far as I can tell, this is the only place in the UI where “Name Suggestion Index” is mentioned. The user might be curious about NSI, but they would have to Google it on their own. Maybe we could provide a link?